### PR TITLE
WIP: Integrate test framework in QC pipeline.

### PR DESCRIPTION
**Pull request by rhpvorderman**

### CODE REVIEW

File: inputs.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
+{
```
Comment (ffinfo) [inline inputs.json]: is this file needed? ;)

File: src/test/scala/biowdl/test/QCBothReads.scala [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/*
```
Comment (ffinfo) [inline src/test/scala/biowdl/test/QCBothReads.scala]: Maybe we can merge some files together?

Comment (rhpvorderman) [reply src/test/scala/biowdl/test/QCBothReads.scala]: Every test a single file is quite nice. Are you referring to package objects?
